collect_app/src/main/res/xml/form_management_preferences.xml

```diff
@@ -48,4 +48,20 @@
             android:summary="@string/instance_sync_summary"
             android:title="@string/instance_sync" />
     </PreferenceCategory>
+
+    <PreferenceCategory
```

Comment:
I'd move this up before form submission since it's so important.

Reply:
Fixed.